### ISSUE TITLE
build under current npm fails in many regards

### ISSUE THREAD

**Issue by trailsandtribulations** [brianc/node-libpq#26]
have been using node-libpq, but now can no longer build with npm.  seems to be two sorts of errors:

<pre>
../node_modules/nan/nan.h:324:27: error: redefinition of ‘template<class T> v8::Local<T> Nan::imp::NanEnsureHandleOrPersistent(const v8::Local<T>&)’
   NAN_INLINE v8::Local<T> NanEnsureHandleOrPersistent(const v8::Local<T> &val) {
</pre>

and

<pre>
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(char*, size_t, int)’:
../node_modules/nan/nan.h:761:50: error: ‘callback’ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                  ^
../node_modules/nan/nan.h:761:60: error: ‘hint’ was not declared in this scope
</pre>

**Comment by bkw** [comment]
This should be fixed once #31 is merged.

**Comment by brianc** [comment]
Should be fixed thanks to @bkw - can you test again @trailsandtribulations?

**Comment by trailsandtribulations** [comment]
will test and reply

**Comment by trailsandtribulations** [comment]
compiles!

if functional problems are found, which I doubt, will post new issue.
